adal/tests/Test.ADAL.NET.UIAutomation/XamarinDroidADALTests.cs

```diff
@@ -47,7 +47,7 @@ public void AcquireTokenInteractiveTestADAL()
         /// <summary>
         /// Runs through the standard acquire token silent flow
         /// </summary>
-        [Test]
+        //[Test]
```

Comment:
please ignore, this is just for app center testing

Reply:
? ok... 

---
In reply to: [219612532](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1235#discussion_r219612532) [](ancestors = 219612532)